### DISCUSSION TITLE
训练时准确率几乎接近1，但是预测时准确率不到0.8

### DISCUSSION THREAD

**Discussion by Lebron-Harden** [PaddlePaddle/PaddleOCR#12177] [Q&A]
![image](https://user-images.githubusercontent.com/74600923/135754453-20acc8cd-a6fe-425b-9260-4052d9a165db.png)
![image](https://user-images.githubusercontent.com/74600923/135754465-3bd8f1ac-a54e-4113-ae59-1e14700bf76e.png)
rt，我用了一个古籍数据集训练，模型是ResNet34+Bilstm+CTC，训练的效果很好，但预测起来就是不太行，请问具体是什么原因呢？yml文件的一些超参数如下：
![image](https://user-images.githubusercontent.com/74600923/135754540-d9e73838-4ad3-48d1-9a9e-e0321767efc7.png)

**Comment by ioracion** [comment]
看起来是过拟合

**Comment by Lebron-Harden** [comment]
> 看起来是过拟合

应该不是，训练的时候，训练准确率一直比测试准确率高十几个点

**Comment by LDOUBLEV** [comment]
> 应该不是，训练的时候，训练准确率一直比测试准确率高十几个点

这个是有点过拟合了，建议加一些数据增强，或者分析下预测错误的是哪些文字，针对性分析下问题

**Comment by Lebron-Harden** [comment]
> > 应该不是，训练的时候，训练准确率一直比测试准确率高十几个点
> 
> 这个是有点过拟合了，建议加一些数据增强，或者分析下预测错误的是哪些文字，针对性分析下问题

请问可以用一些合成数据增强吗?

**Comment by haooxia** [comment]
请问微调之后识别率有所改善吗？然后您使用的是啥古籍数据集呀

**Comment by Lebron-Harden** [comment]
> 请问微调之后识别率有所改善吗？然后您使用的是啥古籍数据集呀

我后来换了模型了（捂脸），古籍数据集叫MTHv2，好像是华南理工大学建的
